src/base-components/Listing/BaseListing.js

```diff
@@ -268,16 +276,23 @@ export default {
             axios.get(this.url, options).then(response => this.populateCurrentStateAndData(response.data.data), error => {
                 // TODO handle error
             });
+
+            if(updateUrl) {
+                this.updateUrl(options.params);
+            }
         },
 
-        filter(column, value) {
+        //FIXME: filter can be called by child listing components on create to set default filters
```

Comment:
nerozumiem tomu fixme

Reply:
filter je funkcia ktoru mozu child componenty volne volat, problemy to robi ked volaju filter na create componentu (prvotny load stranky). Toto mozu robit komponenty, ktore chcu nastavit defaultne filtre. V takom pripade nam to pokazi tuto funkcionalitu, lebo v podstate nam tym prestavia filtre, ktore by sa inak mali brat z urlky.

O tom je ten FIXME, ale pravdepodobne nema dobre riesenie okrem zapisu do nejakeho upgrade guidu.

Reply:
a neda sa introducnut nejaka nova metodka, aby sme nepouzivalu taku, ktora je pravdepodobne overridden? nejak inak sa to tam neda hooknut?